src/app/pages/apps/components/available-apps/available-apps.component.ts

```diff
@@ -66,16 +82,61 @@ export class AvailableAppsComponent implements OnInit, AfterViewInit {
     return `${app.catalog.id}-${app.catalog.train}-${app.name}`;
   }
 
-  private loadTestData(): void {
-    // TODO: Temporary
+  trackByAppSectionTitle(_: number, appSection: AppSection): string {
+    return `${appSection.title}`;
+  }
+
+  private loadApplications(): void {
     this.loader.open();
-    this.appService.getAllCatalogs().pipe(
-      catalogToAppsTransform(),
-      untilDestroyed(this),
-    ).subscribe((apps) => {
-      this.apps = apps;
-      this.loader.close();
-      this.cdr.markForCheck();
+
+    combineLatest([this.appService.getAllApps(), this.appService.getAllAppsCategories()])
+      .pipe(untilDestroyed(this))
+      .subscribe(([apps, appCategories]) => {
+        this.setupApps(apps, appCategories);
+        this.loader.close();
+        this.cdr.markForCheck();
+      });
+  }
+
+  private setupApps(apps: CatalogApp[], appCategories: string[]): void {
+    this.apps = apps;
+
+    this.allRecommendedApps = this.apps.filter((app) => app.recommended);
+    this.allNewAndUpdatedApps = this.apps
+      .sort((a, b) => new Date(a.last_update).getTime() - new Date(b.last_update).getTime());
+
+    this.recommendedApps$.next(this.allRecommendedApps.slice(0, this.sliceAmount));
+    this.newAndUpdatedApps$.next(this.allNewAndUpdatedApps.slice(0, this.sliceAmount));
+
+    this.appSections.push(
+      {
+        title: this.translate.instant('Recommended Apps'),
```

Comment:
Let's add a `category` field in these objects to easily identify which categories this section to belong to. For recommended apps use `recommended` and for new and updated apps use `new` or something similar. In the future when we implement the filter, we can add these two options as checkboxes to only show those apps on the filtered results if the use wants to filter by those. We can decide this after discussion.